src/js/components/StatusIcon.js

```diff
@@ -0,0 +1,56 @@
+import React, { PropTypes, Component } from "react";
+
+import Icon from "#SRC/js/components/Icon";
+
+const ICONS_STATE = {
```

Comment:
Since these states are also used in `ServiceStatusIcon` I think there should this should be moved to a constants file.

Reply:
Not really used in `ServiceStatusIcon` because in `ServiceStatusIcon` we use the `serviceStatus` constants and here I only use the same state convention so we don't need to map serviceStatus to statusIcon. that being said only `transition` isn't a mapped state. 

I'm fine moving to constants though but from my conversation with Orlando and Matt looks like this is ideal since isn't a good idea to always use constants.

Reply:
Okay, I just referring to this in particular: https://github.com/dcos/dcos-ui/pull/2475/files#diff-357cc9e62242bdfbc72ddfa7eaa929afR109 hardcoded constant string.

Reply:
Agree @ahoskins I'm indifferent using a separated file as constants or same file constants.

I could use the same argument though for many other cases where we hardcoded strings in our code base and potentially could be a separated constant file.

I believe @orlandohohmeier will have feedback on this case.

Reply:
Actually, @Poltergeist brought a great point and I'm moving it to constants. https://github.com/dcos/dcos-ui/pull/2475/files#r148768468